gokart/run.py

```diff
@@ -86,19 +86,20 @@ def _try_to_send_event_summary_to_slack(slack_api: Optional[gokart.slack.SlackAP
     if slack_api is None:
         # do nothing
         return
+    options = gokart.slack.SlackConfig()
     with CmdlineParser.global_instance(cmdline_args) as cp:
         task = cp.get_task_obj()
-        tree_info = gokart.make_tree_info(task, details=True)
+        tree_info = gokart.make_tree_info(task, details=True) if options.send_tree_info else ''
         task_name = type(task).__name__
 
     comment = f'Report of {task_name}' + os.linesep + event_aggregator.get_summary()
     content = os.linesep.join([
         '===== Event List ====',
         event_aggregator.get_event_list(),
         os.linesep,
-        '==== Tree Info ====',
-        tree_info,
     ])
+    if options.send_tree_info:
+        content += os.linesep.join(['==== Tree Info ====', tree_info])
```

Comment:
In this code, the linesep between content and tree_info might be eliminated.
How about 

```suggestion
        content = os.linesep.join([content, '==== Tree Info ====', tree_info])
```

or run `os.linesep.join` at the very end.

Reply:
or... might there be redundant linesep in original code?

Reply:
Thank you. I fixed it.